main.py

```diff
@@ -72,10 +71,11 @@ def main():
     files = get_file_list()
     new_files = get_files(files, DL_FILE_NAME)
     if new_files:
-        print 'Preparing email to send new files'
+        print ('Preparing email to send new files and Preparing sms to notify')
         prepare_email(new_files)
+        prepare_sms(new_files)
```

Comment:
Can you make this optional, please?
Add a flag to config `NOTIFY_VIA_SMS` with `false` as default.
Then we can do 
```python
if NOTIFY_VIA_SMS:
        prepare_sms(new_files)
```

We should do this for the emails also https://github.com/jarifibrahim/resultBot/issues/6